src/test/run-make/pass-linker-flags-upwork/Makefile

```diff
@@ -0,0 +1,12 @@
+-include ../../run-make-fulldeps/tools.mk
+
+flag_1 = some_flag
+
+all:
+	# Build deps
+	$(RUSTC) dep_1.rs --crate-type=staticlib
+	$(RUSTC) dep_2.rs --crate-type=staticlib
+	$(RUSTC) lib.rs -l static:-bundle=dep_1 -llink-arg=${flag_1} -l static:-bundle=dep_2 --crate-type=lib
```

Comment:
Could you make the names a bit more obvious, something like `dep_1,2` -> `nativedep_1,2`, `lib` -> `rustdep`.